bridges/ethereum/examples/post_dr.rs

```diff
@@ -0,0 +1,83 @@
+use log::*;
+use std::{
+    sync::Arc,
+    time::{SystemTime, UNIX_EPOCH},
+};
+use web3::{
+    contract,
+    futures::{future, Future},
+    types::U256,
+};
+use witnet_data_structures::{chain::DataRequestOutput, proto::ProtobufConvert};
+use witnet_ethereum_bridge::{
+    config::{read_config, Config},
+    eth::EthState,
+};
+
+fn data_request_example() -> DataRequestOutput {
+    let start = SystemTime::now();
+    let since_the_epoch = start
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards");
+
+    let build_dr_str = include_str!("../../../examples/bitcoin_price.json");
+    let build_dr: serde_json::Value = serde_json::from_str(build_dr_str).unwrap();
+    let mut data_request_output: DataRequestOutput =
+        serde_json::from_value(build_dr["params"]["dro"].clone()).unwrap();
+    data_request_output.data_request.not_before = since_the_epoch.as_secs();
+
+    data_request_output
+}
+
+fn eth_event_stream(
```

Comment:
I missed small comments describing what the function is doing and some small description of the used values (e.g. gas)